src/components/Persona/__stories__/Persona.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+import {faker} from '@faker-js/faker/locale/en';
+import {Persona} from '../Persona';
+
+export default {
+    title: 'Components/Persona',
+    component: Persona,
+} as ComponentMeta<typeof Persona>;
+
+const person = 'Charles Darwin';
+const email = faker.internet.email(...person.split(' '));
+const personImg = faker.internet.avatar();
+
+const Template: ComponentStory<typeof Persona> = (args) => <Persona {...args} />;
```

Comment:
Looks like we missed borders here:

<img width="823" alt="image" src="https://user-images.githubusercontent.com/15698578/229765503-d632420b-a59c-4be7-abfb-5162cb5d010b.png">

Seems prop "theme" doesn't work

Reply:
It's strange, i have borders here

Reply:
Oh, i see it, bug is reproduced in chrome-based browsers, but not firefox